terraform/deployments/github/README.md

```diff
@@ -0,0 +1,23 @@
+# GOV.UK GitHub
```

Comment:
I think this needs a more specific name and a description of what specifically it's for, especially as we already have [another config-as-code tool that's configuring GitHub](https://github.com/alphagov/govuk-saas-config/tree/master/github). We'll also need to signpost the two tools from each other - and from devdocs - otherwise it's gonna be painful for anyone trying to figure out how to change GitHub settings in future.